PageViews/PageViews.js

```diff
@@ -37,6 +38,7 @@ import css from './PageViews.module.less';
 const PageViewsBase = kind({
 	name: 'PageViews',
 	propTypes: /** @lends sandstone/PageViews.PageViewsBase.prototype */ {
+		'aria-label': PropTypes.string,
```

Comment:
Let's update the example with `aria-label`
and add one blank line :)

Reply:
I updated example with `aria-label` and added blank line.